### PR TITLE
Gives Medical professions and Botany unique backpacks and satchels

### PR DESCRIPTION
**Pull request by TheVekter**
I found some nice satchel sprites in the code and found out they were optionals on metastation. A couple folks asked me to port them over as a default for a few jobs.

The following jobs will spawn with a new backpack in their lockers:
- Geneticist
- Chemist
- Virologist
- Botanist
- Scientist

Each job also has a corresponding satchel, if the player prefers. I also fixed the on-mob sprites for a few of the job-specific satchels; they were just using the default instead of the cool colored ones in the dmi.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
All this is telling me is that we need the chameleon backpack to be coded and to come with the chameleon jumpsuit even more now

**Comment by Incoming5643** [comment]
the whole reason they were left option was that it made disguising as roles all the more difficult

**Comment by paprka** [comment]
>medical professions
>botany

also what incoming said, these should remain optional. we tried this on NT and it does make disguising harder when everyone has unique backpacks for jobs.

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -124,6 +124,37 @@
...
+	icon_state = "chempack"
+	item_state = "chempack"
+
+/obj/item/weapon/storage/backpack/genetics
+	name = "genetics backpack"
+	desc = "A bag designed to be super tough, just in case someone hulks out on you."
+	icon_state = "genepack"
+	item_state = "genepack"
+
+/obj/item/weapon/storage/backpack/science
```
Comment (paprka) [inline code/game/objects/items/weapons/storage/backpack.dm]: >medical professions

**Comment by TheVekter** [comment]
That's a good point. I'll have them spawn in the lockers instead of forcing them.

**Comment by paprka** [comment]
maybe you could do what bay does. backpack/satchel like it is, with grey satchels or backpacks, and then unique job-specifc options like 'alt backpack' and 'alt satchel' for job specific packs to shoo metagaming.

Then again, i like forcing sec to have cool ass sec backpacks/satchels so

**Comment by TheVekter** [comment]
I'll look into that.

**Comment by WJohn** [comment]
:+1: 
While I know this'll make it even harder for changelings to quickly disguise themselves as their victims, I think the extra recognition we can get out of this for both new and old players is great. The sprites look good, too.

**Comment by Chocobro** [comment]
sprites look detailed and brilliantly shaded, but I think it's too bright. very high contrast.
otherwise :+1:

**Comment by Ikarrus** [comment]
Too gaudy

**Comment by TZK13** [comment]
I've always thought they looked too gaudy as well but there's no real harm in providing them as an option just so long as it isn't the default which is how it is right now.

**Comment by TheVekter** [comment]
@TZK13 I haven't actually coded anything in but the bags yet. They're going to be optional unless I hear enough feedback that people want it default.

**Comment by Anonus** [comment]
Why does this have a changelog when the change isn't actually added?

**Pull request merged**